app/models/organization.rb

```diff
@@ -0,0 +1,5 @@
+class Organization < ApplicationRecord
+    has_many :users
+    has_many :facilities
+    has_many :tanks, through: :facilities
```

Comment:
I _think_ we don't need the `has_many :tanks`, but feel like my understanding of the stakeholder needs is a little too weak to say for sure.